### DISCUSSION TITLE
Avatar: support to image

### DISCUSSION THREAD

**Discussion by buenojosias** [tasteui/tasteui#574] [Ideas]
Currently the avatar component only supports text and placeholder. I miss adding image support to it. Is it possible to add this feature?

**Comment by devajmeireles** [comment]
Sounds like a good idea. I'll work on this! 👍🏻

**Comment by devajmeireles** [comment]
Done! Released as part of v1.34.0: https://github.com/tallstackui/tallstackui/releases/tag/v1.34.0
